src/web/components/elements/EmbedBlockComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { unescapeData } from '@root/src/lib/escapeData';
+import { css } from 'emotion';
+import { textSans } from '@guardian/src-foundations/typography';
+import { text } from '@guardian/src-foundations/palette';
+
+type Props = {
+    html: string;
+    alt?: string;
+};
+
+const emailCaptionStyle = css`
+    ${textSans.xsmall()};
+    word-wrap: break-word;
```

Comment:
Does this degrade okay on IE11? I don't think that browser support `break-word`

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
Have updated to a more supported property.